test/cli/test/custom-functions/policy.yaml

```diff
@@ -17,3 +17,25 @@ spec:
         - key: "{{base64_decode(request.object.data.value)}}"
           operator: NotEquals
           value: "{{request.object.metadata.labels.value}}"
+---
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: pattern-match
+spec:
+  validationFailureAction: enforce
+  background: false
+  rules:
+    - match:
+        all:
+          - resources:
+              kinds:
+                - Namespace
+      name: label-must-match-pattern
+      validate:
+        deny:
+          conditions:
+            all:
+              - key: '{{pattern_match(`"prefix-*"`, `"{{request.object.metadata.labels.value}}"`)}}'
```

Comment:
Very minor, but I think it would be better to invert the quotes here:
`"{{pattern_match('prefix-*', request.object.metadata.labels.value)}}"`

I didn't test this, just looked here for reference:
https://github.com/kyverno/policies/blob/75228cedbaa173ced40f8f19332056ae14dcc40e/cert-manager/limit-duration/limit-duration.yaml#L33

Reply:
Way nicer indeed 😊